hack/release.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source $(dirname $0)/../vendor/github.com/knative/test-infra/scripts/release.sh
+
+# Local generated yaml file
+readonly OUTPUT_YAML=build.yaml
+
+function build_release() {
+  local image_base_path="${KO_DOCKER_REPO}/github.com/knative/observability/cmd"
+  local image_tag=""
+
+  if [[ -n "${TAG:-}" ]]; then
```

Comment:
The release job will automatically tag all images present in your generated manifest (and that are in the destination GCR). So, unless you're running this locally, $TAG will be either X.Y.Z (for named releases) or vYYYYDDMM-commithash.

More details in https://github.com/knative/test-infra/blob/master/scripts/README.md#using-the-releasesh-helper-script

Reply:
It didn't seem like this applied to our own images (aka built from Dockerfiles outside ko), whether running locally or in publish mode. Is there a cleaner way to make our image references kosher?

Reply:
Interesting. It could be a bug in the image name parsing in https://github.com/knative/test-infra/blob/master/scripts/release.sh#L44 . Can I see the manifest file and the script output?

Reply:
I believe that it's because our own images (in the source manifests under `config`) refer to these images as, e.g., `oratos/cert-generator:v0.23.0`, not using the `@sha256:...` syntax that that logic expects. Not sure how to fix that, given that references in that syntax aren't possible until the image is pushed.

Reply:
Makes sense, thanks. I suggest adding a simple comment here about that to avoid future confusion.